tensorflow/lite/kernels/kernel_util.h

```diff
@@ -144,6 +144,17 @@ TfLiteStatus GetIntermediatesSafe(const TfLiteContext* context,
                                   TfLiteTensor** tensor);
 #endif  // TF_LITE_STATIC_MEMORY
 
+inline int ValidateTensorIndexing(const TfLiteContext* context, int index,
+                                  int max_size, const int* tensor_indices) {
```

Comment:
let's have this function not be inlined. I'm happy to make this change in TF -- that might be faster, let me know.